source/views/settings/screens/api-test.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react'
+import * as c from '@frogpond/colors'
+import {Platform, StyleSheet, ScrollView, TextInput, Text} from 'react-native'
+import {Toolbar} from '@frogpond/toolbar'
+import {fetch} from '@frogpond/fetch'
+import {API} from '@frogpond/api'
+import glamorous from 'glamorous-native'
+import {iOSUIKit, material} from 'react-native-typography'
+import type {NavigationScreenProp} from 'react-navigation'
+
+const styles = StyleSheet.create({
+	container: {
+		backgroundColor: c.white,
+		flex: 1,
+	},
+	default: {
+		height: 44,
+		borderBottomWidth: StyleSheet.hairlineWidth,
+		borderColor: c.black,
+		flex: 1,
+		fontSize: 13,
+		paddingVertical: 4,
+		paddingHorizontal: 8,
+	},
+	data: {
+		paddingVertical: 10,
+		paddingTop: 10,
+	},
+})
+
+export const Paragraph = glamorous(Text)({
+	marginVertical: 3,
+	paddingRight: 4,
+	...Platform.select({
+		ios: iOSUIKit.bodyObject,
+		android: material.body1Object,
+	}),
+})
+
+type Props = {
+	navigation: NavigationScreenProp<*>,
+}
+
+type State = {
+	results: string,
+	error: string,
+}
+
+export class APITestView extends React.PureComponent<Props, State> {
+	static navigationOptions = {
+		title: 'API Tester',
+	}
+
+	state = {
+		results: '',
+		error: '',
+	}
+
+	fetchData = async (path: string) => {
+		try {
+			let responseData: Array<any> = await fetch(API(path), {
+				cache: 'no-store',
+			}).json()
+			let stringifiedResults = JSON.stringify(responseData)
```

Comment:
Why not just call .text() instead of json+stringing?

Reply:
That's an excellent suggestion!

Reply:
Also on the previous bit - it won't always be an array. Don't cast it to Array.